Makefile

```diff
@@ -124,7 +124,7 @@ sched_image: scheduler
 	@coreSHA=$$(go list -m "github.com/apache/incubator-yunikorn-core" | cut -d "-" -f5) ; \
 	siSHA=$$(go list -m "github.com/apache/incubator-yunikorn-scheduler-interface" | cut -d "-" -f6) ; \
 	shimSHA=$$(git rev-parse --short=12 HEAD) ; \
-	docker build ./deployments/image/configmap -t ${REGISTRY}/yunikorn-scheduler-k8s:${VERSION} \
+	docker build ./deployments/image/configmap -t ${REGISTRY}/yunikorn:core-${VERSION} \
```

Comment:
I think here we are building the scheduler image not the core. If we really need the rename please update the comment and the target name as well.

Reply:
We need to file a jira to move the main build from the shim to the core. Including this docker build. That requires a lot of work moving the entrypoint and other code from one repo to the other and update the startup code to check for an integrated k8shim. This is not something for 0.9 but it is a blocker for a 1.0 release.

The changes in the name are needed to allow us to use _yunikorn_ as a container under the apache hub account to store all our images